### DISCUSSION TITLE
Integration with Measurements.jl

### DISCUSSION THREAD

**Discussion by KronosTheLate** [JuliaPlots/Makie.jl#4197] [Ideas]
It would be lovely to be able to plot measurements, as defined by `Measurements.jl`, using Makie. I have created [this issue](https://github.com/JuliaPhysics/Measurements.jl/issues/95) over there, becasue I think that `Measurements.jl` has to export the recipe (rather than Makie itself). This issue is created  to get people from Makie also involved with the issue, should it be nessecary.

**Comment by hz-xiaxz** [comment]
Is there anyone interesting in this feature anymore?

**Reply by chriswaudby** [reply]
Yes - would be very useful!

**Reply by KronosTheLate** [reply]
Yup :D

**Reply by icweaver** [reply]
It's beautiful ='] https://github.com/JuliaPhysics/Measurements.jl/pull/174
